webapp/client/src/pages/PersonAHasDisabilityPage.js

```diff
@@ -0,0 +1,80 @@
+import PropTypes from "prop-types";
+import React from "react";
+import { Trans, useTranslation } from "react-i18next";
+import FormHeader from "../components/FormHeader";
+import FormFieldRadio from "../components/FormFieldRadio";
+import StepForm from "../components/StepForm";
+import StepHeaderButtons from "../components/StepHeaderButtons";
+import Details from "../components/Details";
+import { fieldPropType } from "../lib/propTypes";
+
+export default function PersonAHasDisabilityPage({
+  form,
+  fields,
+  prevUrl,
+  stepHeader,
+  numOfUsers,
+}) {
+  const { t } = useTranslation();
+
+  const tbold = function (key) {
+    return <Trans t={t} i18nKey={key} components={{ bold: <b /> }} />;
```

Comment:
Can you explain why we are putting the header intro and the details text in a `<b>` tag? I don't think I get it.

Reply:
In the translations there is a <bold> tag and that tag will be replaced with a <b> (I think this is the react way of doing this?)

Reply:
Yes, but _why_? :)
Why are intro and details wrapped in a `<b>`?
Do we want to make them bold?
-> If yes, let's use `<strong/>` or a styled-component with `font-weight: bold`.
-> If no, what's the goal?